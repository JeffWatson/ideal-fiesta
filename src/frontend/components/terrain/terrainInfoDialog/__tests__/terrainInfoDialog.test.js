import React from 'react';
import { shallow } from 'enzyme';
import TerrainInfoDialog from '../terrainInfoDialog';

const props = {
  onDialogClose: jest.fn(),
  terrain: {
    defenseBonus: 10,
    actionable: true,
    selectable: true,
  },
};

describe('frontend/components/terrain/terrainInfoDialog component', () => {
  it('renders correctly', () => {
    const component = shallow(<TerrainInfoDialog {...props} />);

    expect(component).toMatchSnapshot();
  });
});
