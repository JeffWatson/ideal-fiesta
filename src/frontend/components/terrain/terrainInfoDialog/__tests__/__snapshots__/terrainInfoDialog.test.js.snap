// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`frontend/components/terrain/terrainInfoDialog component renders correctly 1`] = `
<Dialog
  className="default"
  onClose={[Function]}
>
  <div
    className="terrain-info-dialog"
  >
    <div
      className="defense"
    >
      DEF: 
      10
    </div>
  </div>
</Dialog>
`;
